src/foam/u2/crunch/CapabilityStore.js

```diff
@@ -407,6 +408,14 @@ foam.CLASS({
         this.menuDAO.cmd_(this, foam.dao.CachingDAO.PURGE);
         this.menuDAO.cmd_(this, foam.dao.AbstractDAO.RESET_CMD);
       }
+    },
+    {
+      name: 'onResize',
+      code: function() {
+        var scrollWidth = this.document.querySelector(`.foam-u2-crunch-CapabilityStore-feature-column-grid`).scrollWidth;
```

Comment:
If there is more than one this will break. You stored it in the 'ele' variable, so you could access that the do ele.el().scrollWidth;

Reply:
updated. Thanks